projects/angular/src/button/providers/button-group-focus-handler.service.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable, Renderer2 } from '@angular/core';
+
+import { ClrPopoverToggleService } from '../../utils';
+import { FocusService } from '../../utils/focus/focus.service';
+import { FocusableItem } from '../../utils/focus/focusable-item/focusable-item';
+import { Linkers } from '../../utils/focus/focusable-item/linkers';
+import { InitialFocus } from './button-group-focus.enum';
+
+@Injectable()
+export class ButtonGroupFocusHandler {
+  private initialFocus: InitialFocus = InitialFocus.FIRST_ITEM;
+  private _unlistenFuncs: (() => void)[] = [];
+  private _menuOpened: boolean;
+  private buttons: FocusableItem[];
```

Comment:
`private` members are generally put after `public` members of the same kind. So these should go after the `public` properties.